### PR TITLE
Ship

### PR DESCRIPTION
**Pull request by sagemlee**
Added full ship_test.rb and ship.rb. Tests passed. Interaction pattern on project requirements should flow.

### CODE REVIEW

File: lib/ship.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require 'pry'
```
Comment (KieraAllen) [inline lib/ship.rb:1]: Do we have to require 'pry' on class files?

Comment (sagemlee) [reply lib/ship.rb:1]: No I just used it here, but it isnt necessary.

File: lib/ship.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+class Ship
+  attr_reader :name, :length, :health
+
+  def initialize(name, length)
+    @name = name
+    @length = length
+    @health = length
+    @sunk = false
```
Comment (KieraAllen) [inline lib/ship.rb]: You mentioned we might not need this @sunk...?

Comment (sagemlee) [reply lib/ship.rb]: Oh you're right, let me update before we merge.

**Review by KieraAllen — APPROVED** [review]

**Pull request merged**